tests/src/coffee_httpd_integration_tests.rs

```diff
@@ -67,3 +67,241 @@ pub async fn httpd_init_add_remote() {
 
     cln.stop().await.unwrap();
 }
+
+#[tokio::test(flavor = "multi_thread")]
+#[ntest::timeout(560000)]
+pub async fn httpd_add_remove_plugins() {
+    init();
+
+    let mut cln = Node::tmp("regtest").await.unwrap();
+    let lightning_dir = cln.rpc().getinfo().unwrap().ligthning_dir;
+    let lightning_dir = lightning_dir.strip_suffix("/regtest").unwrap();
+    let manager = CoffeeHTTPDTesting::tmp(lightning_dir.to_string())
+        .await
+        .unwrap();
+    log::info!("lightning path: {lightning_dir}");
+    let url = manager.url();
+    log::info!("base url: {url}");
+    let client = reqwest::Client::new();
+
+    // Define the request body to be sent to the /remote/add endpoint
+    let remote_add_request = RemoteAdd {
+        repository_name: "lightningd".to_string(),
+        repository_url: "https://github.com/lightningd/plugins.git".to_string(),
+    };
+
+    // Send the request to add a remote repository
+    let response = client
+        .post(format!("{}/remote/add", url))
+        .json(&remote_add_request)
+        .send()
+        .await
+        .unwrap();
+
+    // Check the response status code, log the body.
+    assert!(response.status().is_success());
+    let body = response.text().await.unwrap();
+    log::info!("/remote/add response: {}", body);
+
+    // Define the request body to be sent to the /show endpoint
+    let show_request = Show {
+        plugin: "helpme".to_string(),
+    };
+
+    // Send a get request to the /show endpoint
+    let response = client
+        .get(format!("{}/show", url))
+        .json(&show_request)
+        .send()
+        .await
+        .unwrap();
+
+    // Check the response status code, log the body.
+    assert!(response.status().is_success());
+    let body = response.text().await.unwrap();
+    log::info!("/show response: {}", body);
+
+    // Parse the response body
+    let response_json: serde_json::Value = serde_json::from_str(&body).unwrap();
+
+    // Extract the `readme` field from the response JSON
+    let readme = response_json["readme"].as_str().unwrap();
+
+    // Assert that the `readme` starts with the expected content
+    assert!(readme.starts_with("# Helpme plugin"));
```

Comment:
```suggestion
    assert!(readme.starts_with("# Helpme plugin"), "{:?}", readme);
```

do always this inside an assert, if it is not a dubug add a Debug inside the struct derive